test/player_test.rb

```diff
@@ -5,5 +5,9 @@
 require 'pry'
 
 class PlayerTest < Minitest::Test
-
+  def test_player_exists
+    play = Player.new("Bob")
+    refute_nil play
+    assert_equal "Bob", play.name
+  end
```

Comment:
No need to touch this file. Suppose I do like the idea that you're testing this since you added functionality to player, but would prefer that neither had changes.